collections/_article/progress-report-state-of-particles.md

```diff
@@ -0,0 +1,94 @@
+
+To address these issues, and more, the `GPUParticles` system has been refactored and expanded upon. Particles can now support not just realistic types of effects, but also more stylized ones with more exaggerated motion. Additionally, a couple of long-requested quality of life features was added.
+
+## The new `GPUParticles`
+
+**NOTE:** These changes should be 100% backwards compatible. If particles look different after updating, please [file a bug report](https://github.com/godotengine/godot/issues/new/choose).
+
+_[Pioneering of this work](https://github.com/godotengine/godot/pull/78851) was kindly sponsored by [PahdoLabs](https://www.pahdolabs.com/)._
+
+### New parameters for `GPUParticles` nodes
+
+Two new parameters were added to `GPUParticles2D`/`GPUParticles3D` nodes:
+
+- `amount_ratio`: from 0 to 1, a percentage of particles compared to the amount that will be emitted.
+- `interp_to_end`: from 0 to 1, allows to interpolate all particles from the same node simultaneously towards the end of their lifetime.
+
+### Changes in `ParticleProcessMaterial`
+
+The main change in the particle material is the big-big refactor of the internal code. The code was split into smaller, reusable functions, which are
+called both from `start()` and `process()`. This ensures better consistency and more maintainability.
+
+Multiple configuration options were added or changed to expand your control over damping, emission, and velocity.
+
+Damping changes include:
+
+- `damping_as_friction`: flag that changes the behavior of damping from a constant
+deceleration to a deceleration based on speed.
+
+Emission changes include:
+
+- `emission_shape_offset`: an offset for the emission shape, in the node's local space.
+- `emission_shape_scale`: a scale of the emission shape, in the node's local space.
+- `inherit_velocity_ratio`: a percentage of the emitter's velocity to inherit on spawn.
+
+Velocity changes include:
+
+- `velocity_pivot`: a pivot point used for calculating radial and orbital velocity.
+- `directional_velocity`: velocity along an axis; requires an XYZ curve.
+- `orbit_velocity`: orbit velocity can now be used without toggling `disable_z`; requires an XYZ curve; axes are in the node's local space.
+- `radial_velocity`: velocity away or towards the velocity pivot.
+- `velocity_limit`: a curve defining a hard limit for the particles' velocity following their lifetime.
+- `scale_over_velocity`: a curve that allows to scale particles on a given axis.
+
+On top of that an optional separation between alpha over life and emission over life was added. Note that emission over lifetime just multiplies the `COLOR` value of the particle by a given value. This needs to be properly read in the display shader.
+
+You can also check out [GH-79527](https://github.com/godotengine/godot/pull/79527), which covers the bulk of these changes.
+
+I can't wait to see all that you'll make with it! In the meantime, enjoy these VFX made by the testers of the PR.
```

Comment:
So @QbieShay wants to add a video by Rensei here. Still waiting on a link or a file with it.